### ISSUE TITLE
Consider adding pixi install commands to feedstock README template

### ISSUE THREAD

**Issue by matthewfeickert** [conda-forge/conda-smithy#2127]
### Comment:

At the moment the feedstock README template gives instructions for how to install and search for conda-forge packages with `conda` and `mamba`

https://github.com/conda-forge/conda-smithy/blob/73c3564e5e6e7f354243699a5a76729dfb1a734e/conda_smithy/templates/README.md.tmpl#L219-L254

I think it would be really nice to also add instructions for how to install and search using `pixi` (https://pixi.sh/). `pixi` has become the default way I interact with conda-forge packages for all my projects except when external legacy situations from my collaborators currently make it infeasible, so I'd be interested in it getting copy-pasteable commands on conda-forge.

I'm not sure if this is the correct place to discuss this, but as this is the repository that builds / rerenders feedstocks I thought I'd start here.

**Comment by baszalmstra** [comment]
See: https://github.com/conda-forge/conda-smithy/pull/1856

**Comment by matthewfeickert** [comment]
Thanks @baszalmstra! 👍 I had only checked the open and closed Issues and not the PRs so I missed this.

**Comment by matthewfeickert** [comment]
👋 @xhochy @isuruf, I know that you're busy and this is a low priority item compared to most other things happening for the core team, but as you've given reviews of PR https://github.com/conda-forge/conda-smithy/pull/1856 would you be willing to weigh in here (or on PR https://github.com/conda-forge/conda-smithy/pull/1856 again) on the possibility of this?

**Comment by isuruf** [comment]
Maybe we should have a page in our website on different ways of installing conda-forge packages and link to it from the feedstock. It's not going to be feasible to add more and more tools.

**Comment by matthewfeickert** [comment]
> Maybe we should have a page in our website on different ways of installing conda-forge packages and link to it from the feedstock.

Would that mean that there would be no install instructions at all on the feedstocks? That seems strange to me.

**Comment by isuruf** [comment]
The website would have the install instructions there for the packages.

**Comment by matthewfeickert** [comment]
> The website would have the install instructions there for the packages.

Yeah, I understand, but still seems a bit strange. That being said, I think having install instructions that don't favor one tool in particular on a secondary site it still better than having nothing at all.
